third_party/opa/data-and-policies/user_policies.rego

```diff
@@ -42,12 +42,14 @@ transform[action] {
     not dp.check_destination([dp.GeoDestinations.US])
     
     column_names := dp.column_with_any_tag(["SPI", "SMI"])
-    action = dp.build_encrypt_column_action(column_names[_], dp.build_policy_from_description(description))
+    #action = dp.build_encrypt_column_action(column_names[_], dp.build_policy_from_description(description))
+    action = dp.build_redact_column_action(column_names[_], dp.build_policy_from_description(description))
```

Comment:
This policy is not related to the example (with tag = Finance). Why is it changed?

Reply:
Since we don't support encrypt enf action yet I have made redact action as the default transformation. Please let me know if this is ok. Thanks.

Reply:
OK.